be/src/exec/es_http_scan_node.cpp

```diff
@@ -166,6 +166,11 @@ Status EsHttpScanNode::start_scanners() {
 }
 
 Status EsHttpScanNode::collect_scanners_status() {
+    // NOTE. if open() was called, but set_range() was NOT called for some reason.
```

Comment:
This comment seems not related with this PR?

Reply:
Yes. it's last pr i commited. it was committed failed at that time